app/views/beaches/show.html.erb

```diff
@@ -0,0 +1,4 @@
+<% @reviews.each do |review| %>
+<%= review.rating %>
+<%= review.description %>
+<% end %>
```

Comment:
indentation:

```ruby
<% @reviews.each do |review| %>
  <%= review.rating %>
  <%= review.description %>
<% end %>
```

Though this doesn't matter that much because it's a placeholder for now, just fyi.